server/methods.js

```diff
@@ -759,5 +759,35 @@ Meteor.methods({
     }
     console.log("OK");
     return true;
+  },
+  instanceAdminCheck(instanceid, adminId) {
+    let user;
+    const table = Instances.findOne({ _id: instanceid });
+    const adminOfInstance = Meteor.users.findOne({'services.password.reset.email': table.admin});
+    if (table === undefined) { return false; }
+    if (!this.userId) {
+      return false;
+    }
+    if (adminId) {
+      user = Meteor.users.findOne({ _id: adminId });
+    } else {
+      return false;
+    }
+    console.log(user.emails[0].address, table.admin );
+    return (user.emails[0].address === table.admin );
+  },
+  emphasize(divId, instanceId, currUser) {
```

Comment:
@singhsanket143 https://github.com/berkmancenter/question_tool/pull/134/files#r316066171

the same here